resources/monitoring/charts/grafana/values.yaml

```diff
@@ -717,12 +705,6 @@ imageRenderer:
   enabled: false
   replicas: 1
   image:
-    # image-renderer Image repository
-    repository: grafana/grafana-image-renderer
-    # image-renderer Image tag
```

Comment:
we are not using that image/functionality, please just comment out and do not specify in the listing. Please revert related changes of defining the image to have it more easy to update the chart

Reply:
The image was restored